marklogic-data-hub/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
     id 'java'
     id 'maven-publish'
     id 'com.jfrog.bintray' version '1.7.2'
-    id 'com.marklogic.ml-gradle' version '3.1-alpha4'
+    id 'com.marklogic.ml-gradle' version '3.2.0'
```

Comment:
To @aebadirad 's point.. we plan to update to the latest ml-gradle in the next sprint. Can you please set this back to 3.1-alpha4 for now?